roseau/load_flow/models/connectables.py

```diff
@@ -77,11 +77,7 @@ def __init__(
         self._res_currents: ComplexArray | None = None
 
     def __repr__(self) -> str:
-        args = [
-            f"id={self.id!r}",
-            f"bus={repr(self.bus.id) if self.bus is not None else '<disconnected>'}",
-            f"phases={self.phases!r}",
-        ]
+        args = [f"id={self.id!r}", f"bus={self.bus.id!r}", f"phases={self.phases!r}"]
```

Comment:
The `self.bus.id` will fail is the load is disconnected as long as the `bus` property retrieves `None` in this case.